contrib/setup-win32.nsi

```diff
@@ -0,0 +1,90 @@
+#!Nsis Installer Command Script
+#
+# To build an installer from the script you would normally do:
+#
+#   dnf install mingw32-nsis
+#   makensis setup-win32.nsi
+
+Name ""
+OutFile "fwupd-@FWUPD_VERSION@-setup-x86_64.exe"
+InstallDir "$ProgramFiles\fwupd"
+InstallDirRegKey HKLM SOFTWARE\fwupd "Install_Dir"
+ShowInstDetails hide
+ShowUninstDetails hide
+XPStyle on
+Page directory
+Page instfiles
+
+ComponentText "Select which optional components you want to install."
+
+DirText "Please select the installation folder."
+
+Section "fwupd"
+  SectionIn RO
+
+  SetOutPath "$INSTDIR\bin"
+
+  # deps
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/iconv.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libarchive-13.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libbrotlicommon.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libbrotlidec.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libbz2-1.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libffi-6.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libgcc_s_seh-1.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libgio-2.0-0.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libglib-2.0-0.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libgmodule-2.0-0.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libgmp-10.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libgnutls-30.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libgnutls-30.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libgobject-2.0-0.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libgusb-2.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libhogweed-4.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libidn2-0.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libintl-8.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libjson-glib-1.0-0.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/liblzma-5.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libnettle-6.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libp11-kit-0.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libpcre-1.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libpsl-5.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libsoup-2.4-1.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libsqlite3-0.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libtasn1-6.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libunistring-2.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libusb-1.0.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libwinpthread-1.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libxml2-2.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/zlib1.dll"
```

Comment:
Can you use a glob on these dlls too?

Reply:
I could, but we only need those. Including all of them adds another ~40Mb to the bundle...

Reply:
Definitely makes sense to hardcode list then agree.